MozillaFoundation/foundation.mozilla.org
# Main navigation `Blog` dropdown aligned to right

# Description

This PR updates the `Blog` dropdown contents alignment on the main nav, in order to display it on the right side of the screen on desktops.

Current state:
<img width="1402" alt="image" src="https://github.com/user-attachments/assets/8ed034e8-57cc-4a1c-acd6-a6741b7f7bcb">

Updated state:
<img width="1423" alt="image" src="https://github.com/user-attachments/assets/3d21c557-0fda-442f-b344-b0e7ce48723e">

# To Test

Browse the homepage on the [preview app](https://foundation-s-tp1-604-ma-ekwlix.herokuapp.com/en/), hover the main navigation `Blog` element and notice the dropdown content being right aligned.

Link to sample test page: https://foundation-s-tp1-604-ma-ekwlix.herokuapp.com/en/
Related PRs/issues: #12288

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-955)
